apps/preview/next/pages/showcases/Banners/Hero.tsx

```diff
@@ -5,38 +5,28 @@ import { SfButton } from '@storefront-ui/react';
 
 export default function Hero() {
   return (
-    <div className="relative min-h-[576px]">
-      <picture>
-        <source srcSet="http://localhost:3100/@assets/hero-bg.png" media="(min-width: 768px)" />
+    <div className="relative h-dvh w-dvh">
+      <picture className="before:absolute before:inset-0 before:bg-primary-900 before:-z-10">
+        <source srcSet="http://localhost:3100/@assets/hero-bg-2.png" media="(min-width: 768px)" />
         <img
-          src="http://localhost:3100/@assets/hero-bg-mobile.png"
-          className="absolute w-full h-full z-[-1] object-cover"
+          src="http://localhost:3100/@assets/hero-bg-mobile-2.png"
+          className="absolute w-full h-full z-[-1] object-cover opacity-50"
+          alt="hero"
         />
       </picture>
-      <div className="md:flex md:flex-row-reverse md:justify-center min-h-[576px] max-w-[1536px] mx-auto">
-        <div className="flex flex-col md:basis-2/4 md:items-stretch md:overflow-hidden">
-          <img
-            src="http://localhost:3100/@assets/hero-headphones.png"
-            alt="Headphones"
-            className="h-full object-cover object-left"
-          />
-        </div>
-        <div className="p-4 md:p-10 md:flex md:flex-col md:justify-center md:items-start md:basis-2/4">
-          <p className="typography-text-xs md:typography-text-sm font-bold tracking-widest text-neutral-500 uppercase">
-            Feel the music
-          </p>
-          <h1 className="typography-display-2 md:typography-display-1 md:leading-[67.5px] font-bold mt-2 mb-4">
-            New Wireless Pro
-          </h1>
-          <p className="typography-text-base md:typography-text-lg">
-            Spatial audio. Adjustable ear cups. On-device controls. All-day battery.
-          </p>
-          <div className="flex flex-col md:flex-row gap-4 mt-6">
-            <SfButton size="lg"> Order now </SfButton>
-            <SfButton size="lg" className="bg-white" variant="secondary">
-              Show more
-            </SfButton>
-          </div>
+      <div className="h-full w-full max-w-[1536px] mx-auto p-4 md:px-10 flex flex-col justify-end md:justify-center text-white">
+        <p className="typography-text-xs md:typography-text-sm font-bold tracking-widest uppercase">Feel the music</p>
+        <h1 className="typography-display-2 md:typography-display-1 md:leading-[67.5px] font-bold mt-2 mb-4">
+          New Wireless Pro
```

Comment:
copy is not changed

Reply:
good catch, button was wrong as well: 2ba296a338084f1be60136886514e3ce73e22a77